src/rpc/blockchain.cpp

```diff
@@ -781,7 +781,8 @@ static RPCHelpMan getblockfrompeer()
         "getblockfrompeer",
         "\nAttempt to fetch block from a given peer.\n"
         "\nWe must have the header for this block, e.g. using submitheader.\n"
-        "\nReturns {} if a block-request was successfully scheduled\n",
+        "Subsequent calls for the same block and a new peer will cause the response from the previous peer to be ignored.\n"
+        "\nReturns an empty JSON object if a block-request was successfully scheduled.",
         {
             {"blockhash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The block hash"},
             {"nodeid", RPCArg::Type::NUM, RPCArg::Optional::NO, "The node ID (see getpeerinfo for node IDs)"},
```

Comment:
maybe `peer_id`?  the call is named getblockfrompeer, the help description uses "peer", and snakecase is current naming style for rpc args
```suggestion
            {"peer_id", RPCArg::Type::NUM, RPCArg::Optional::NO, "The peer ID (see getpeerinfo for node IDs)"},
```

Reply:
Done, and also changed the variable names.

Reply:
Nice. May need to update client.cpp as well: `{ "getblockfrompeer", 1, "nodeid" },`

Reply:
Oops, fixed.